### DISCUSSION TITLE
Adding multiple aeordynamic inequality constraints

### DISCUSSION THREAD

**Discussion by usmanshahid90** [mdolab/dafoam#150] [General]
Hey, 

I would like to add two aerodynamic constraints i.e.
1)   I would like my Cl to be greater than equal to (let say) 0.7
2)  At the same time, I would like to make my Cm to be equal to a particular value.

How can I perform this on DAFOAM.
 Kindly guide me.

**Comment by friedenhe** [comment]
Check this [case](https://github.com/DAFoam/tutorials/blob/master/Rotor37_Compressor/runScript.py) for setting multiple constraints. I have deleted a duplicated thread.

**Reply by usmanshahid90** [reply]
Thanks for your reply. this page only covers the case for multiple constraints, but inequality constraints aren't there..
can you please guide me about inequality constraints?

**Reply by friedenhe** [reply]
Change the lower and upper bounds.

**Reply by usmanshahid90** [reply]
Aren' those bounds only associated with geometric constraints? 
I'd like to give inequality constraint of coefficient of lift.

**Reply by friedenhe** [reply]
Which bounds are you referring to? Can you copy the line where the lift constraint is added to here? Let me know which part is not clear.

**Reply by usmanshahid90** [reply]
In runscript.py , CL target is defined. In global parameters, It is an equality constraints that the optimized airfoil should have CL= CL_target. I would like to convert it into inequality constraint, i.e. CL_target> 0.7 let say.
I want my optimized airfoil to not restric CL to a particular value, instead, I'd like it to achieve the CL above a threshold to as much as it can.
Regards,

**Reply by friedenhe** [reply]
You are not looking at the right line, search for "CL_target" in runScript.py. It is used somewhere else.
